src/main/resources/db/migration/R__Comments.sql

```diff
@@ -65,68 +65,62 @@ COMMENT ON TABLE device_templates IS 'Canned device configurations for use in ca
 
 COMMENT ON TABLE devices IS 'Hardware devices managed by the device manager at a facility.';
 
-COMMENT ON TABLE document_saved_versions IS 'Saved versions of document variable values. A saved version is conceptually just a reference to a particular point in the edit history of the document; to restore that version, we ignore any later edits.';
+COMMENT ON TABLE docprod.document_saved_versions IS 'Saved versions of document variable values. A saved version is conceptually just a reference to a particular point in the edit history of the document; to restore that version, we ignore any later edits.';
```

Comment:
Can we move these down to their own section of this file like we do for the nursery and accelerator and tracking tables? (At some point we should do the same for the seedbank tables so they aren't jumbled together with the public ones.)

Reply:
This will be addressed in another PR into the feature branch, for now it is captured as a TODO in the main feature branch PR description.